etcdserver/v3_server.go

```diff
@@ -85,14 +86,19 @@ type Authenticator interface {
 }
 
 func (s *EtcdServer) Range(ctx context.Context, r *pb.RangeRequest) (*pb.RangeResponse, error) {
+	trace := traceutil.New("Range")
```

Comment:
Once we start using this trace output for something real, I'm betting we will immediately want to know more about the request that was traced. What keys were requested? How many response KVs were returned?

Reply:
We can cross compare the total time in trace output vs. the total time in expensive request warning output.

Reply:
That helps for information available in expensive request warning. To correlate a request identifier of some sort that is printed on both the request warnings and the traces would be more developer friendly than the latency correlation.

I suspect we're going to need to print out various parameters for the trace as a whole and for trace steps. Some of which might not already be available in the expensive request warning.